sormas-backend/src/main/resources/sql/sormas_schema.sql

```diff
@@ -11659,4 +11659,13 @@ INSERT INTO userroles_userrights (userrole_id, userright) SELECT userrole_id, 'U
 
 INSERT INTO schema_version (version_number, comment) VALUES (473, 'Add user roles view to UI #4462');
 
+-- 2022-07-25 Make region and district required for aggregate reports
+DELETE FROM aggregatereport
+WHERE region_id IS NULL OR district_id IS NULL;
+
+DELETE FROM aggregatereport_history
```

Comment:
I'd keep the history table entries just in case anyone wants to restore data that is deleted with the script above; @MartinWahnschaffe what do you think?

Reply:
Deleting from the history table is redundant, since we already have a trigger that takes care of this.
If we want to create a backup of the deleted data we would have to deactivate this trigger and then create a trigger that creates the history entry on deletion. From my point of view this would not be worth it, but we need a bold comment in the release notes that mentions this. A backup should be created when updating the system anyway.

Reply:
the code for deletion of aggregatereport_history table was removed